app/src/components/LoginRegister/Auth0Landing.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from "react";
```

Comment:
From what I can tell, this file has a path of `/auth0-landing` specified in the LoginRegister/index component but it is never being linked to from the UI. 

I can see that in the `choose` component, you are instantiating an auth0 client and logging in with auth0 directly on button click which seems accurate but it is not then sending the token to the BFF client which seems like a required thing to do, so I'm a bit confused as to the existence of this file and auth0 being instantiated in both places.

Reply:
Ignore this comment as I've just realised that this is the redirect URL after authentication with auth0.